index.js

```diff
@@ -1,5 +1,31 @@
 'use strict';
 var chalk = require('chalk');
+var isSupported = (function() {
+	if (process.platform !== 'win32') {
+		return true;
+	}
+
+	var env = process.env;
+
+	if (env.APPVEYOR) {
+		return true;
+	}
+
+	if (env.ConEmuDir) {
```

Comment:
![image](https://user-images.githubusercontent.com/1171691/28267683-b6d7add8-6b2d-11e7-8de1-8da68990e695.png)
[cmder](http://cmder.net/) is a very popular CMD substitute on windows, it is run base ConEmu.
and ConEmu not support info and warning icons.